lineage/Analyze.py

```diff
@@ -164,33 +163,42 @@ def Results(tHMMobj, pred_states_by_lineage, LL):
     results_dict["switcher_map"] = switcher_map
 
     # Rearrange the values in the transition matrix
-    temp_T = tHMMobj.estimate.T
+    temp_T = np.zeros((tHMMobj.num_states,tHMMobj.num_states))
```

Comment:
The other time where I copy the estimate T and then change the copy, but this also changes the original. Now it uses a matrix that is no where related to the original copy, so changing it doesn't affect the original.